nginx.conf

```diff
@@ -1,10 +1,21 @@
-worker_processes      1;
+worker_processes      auto;
+
+error_log  /var/log/nginx/error.log notice;
```

Comment:
Can you collaborate why changing the `error_log`?

Reply:
this ist just a merge of your your configuration an the one from the `nginxinc/nginx-unprivileged:1.21.6-alpine` image